src/executor/graph_executor.cc

```diff
@@ -1360,6 +1359,8 @@ void GraphExecutor::InitOpSegs() {
   if (prefer_bulk_exec_inference && !is_training) {
     this->BulkInferenceOpSegs();
   }
+
+  return;
```

Comment:
why do we need a return at the end of a function?

Reply:
It looks cool